src/rebuild/srv.c

```diff
@@ -790,16 +795,12 @@ rebuild_prepare(struct ds_pool *pool, uint32_t rebuild_ver,
 		struct rebuild_global_pool_tracker **rgt)
 {
 	pool_comp_state_t	match_status;
-	unsigned int		master_rank;
 	int			rc;
 
 	D_DEBUG(DB_REBUILD, "pool "DF_UUID" create rebuild iv, op=%s\n",
 		DP_UUID(pool->sp_uuid), RB_OP_STR(rebuild_op));
 
 	/* Update pool iv ns for the pool */
-	crt_group_rank(pool->sp_group, &master_rank);
-	ds_pool_iv_ns_update(pool, master_rank);
```

Comment:
sorry, why drop the code? need not update the master_rank?

Reply:
oh, iv ns update should be done through iv pool map update, instead of by rebuild iv, just cleanup a bit.